src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -22,7 +22,7 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
 
         public Task<IEnumerable<KeyValuePair<string, string>>> ProcessKeyValue(ConfigurationSetting setting, Logger logger, CancellationToken cancellationToken)
         {
-            FeatureFlag featureFlag = ParseFeatureFlag(setting.Key, setting.Value);
+            FeatureFlag featureFlag = ParseFeatureFlag($"Failed to parse invalid feature flag json. Key: '{setting.Key}'", setting.Value);
```

Comment:
Is this intentional?

Reply:
This looks weird now that I see it again, the reason I did it was `Setting.key` was only being used for error logging but it should definitely be a different variable name or a class variable.

Reply:
Let's pass the key and log the message wherever its needed. I thought this was accidentally copy/pasted :)